pkg/config/env_var.go

```diff
@@ -84,15 +84,20 @@ func NewEnvVarListFromSlice(envList []string) (EnvVarList, error) {
 
 // RemoveEnvVarsFromList removes the env variables based on the keys provided
 // and returns a new EnvVarList
-func RemoveEnvVarsFromList(envVarList EnvVarList, keys []string) EnvVarList {
+func RemoveEnvVarsFromList(envVarList EnvVarList, keys []string) (EnvVarList, error) {
+	envVarInList := false
```

Comment:
This will stay false if the keys is an empty list and hence throw error

Reply:
@dharmit add a check for the `keys` length. If its 0 then we return the same list without any error